Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalTrg_cff.py

```diff
@@ -4,6 +4,7 @@
 ecalTrgHLT = HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
 #    eventSetupPathsKey='AlCa_EcalPhiSym*', # this is the HLT path that can be used                                                                                                          
      eventSetupPathsKey='EcalTrg',
+     throw = cms.bool( False ),
```

Comment:
See the discussion at the equivalent 93X PR #19510: 
https://github.com/cms-sw/cmssw/pull/19510#issuecomment-312589682